app.py

```diff
@@ -21,16 +21,15 @@ def str_to_bool(str_input):
 
 
 # Load environment variables
-azure_openai_endpoint = os.environ.get("AZURE_OPENAI_ENDPOINT")
-azure_openai_key = os.environ.get("AZURE_OPENAI_KEY")
 openai_api_key = os.environ.get("OPENAI_API_KEY")
-authentication_required = str_to_bool(os.environ.get("AUTHENTICATION_REQUIRED", False))
-assistant_id = os.environ.get("ASSISTANT_ID")
 instructions = os.environ.get("RUN_INSTRUCTIONS", "")
-assistant_title = os.environ.get("ASSISTANT_TITLE", "Assistants API UI")
 enabled_file_upload_message = os.environ.get(
     "ENABLED_FILE_UPLOAD_MESSAGE", "Upload a file"
 )
+openai_assistants = json.loads(os.environ.get("OPENAI_ASSISTANTS"))
```

Comment:
If you did not set OPENAI_ASSISTANTS, please maintain the traditional UI.
There are cases where you want to operate with one assistant.